### ISSUE TITLE
question about uw

### ISSUE THREAD

**Issue by WBS-123** [median-research-group/LibMTL#40]
In the uw original paper, the objective function is:
![image](https://github.com/median-research-group/LibMTL/assets/49219230/d696bca2-64ed-4c74-805b-5e2391a2d1b2)
and according to the paper,
![截屏2023-05-16 15 53 12](https://github.com/median-research-group/LibMTL/assets/49219230/3c032f54-394d-48ea-859d-0c290b1b6f8c)
the second item ![截屏2023-05-16 16 12 34](https://github.com/median-research-group/LibMTL/assets/49219230/caeba7a6-77d8-404e-9b54-d50d3b261230) is diffirent with first item ![截屏2023-05-16 16 13 44](https://github.com/median-research-group/LibMTL/assets/49219230/43c4ed6b-ab8e-4c02-b31c-f6da7abae53c) in denominators.
But in you code, loss = (losses/(2*self.loss_scale.exp())+self.loss_scale/2).sum(), without distinction between the denominators of these two.
Is that correct?

**Comment by Baijiong-Lin** [comment]
The implementation in our code is according to Eq. (7) in the original paper, which is quite general since it just needs an assumption that each task output follows a Gaussian distribution and it does not need to know what type of loss function for each task. In some cases, you can deduce it again according to the loss functions you use (like Eq. (10) in the original UW paper).
